### PR TITLE
Small fixes

### PR DESCRIPTION
**Pull request by ghost**
API Fixes:
- user is set correct when create/update record
- validate error is thrown when record has bad `name` or `content`

### CODE REVIEW

**Comment by szok** [comment]
:ok:

**Comment by szok** [comment]
:ok:

File: dnsaas/api/v2/tests.py [hunk 1]
```diff
@@ -284,6 +305,29 @@ def test_dont_reject_update_when_request_already_exists_but_superuser(
...
+        self.client.login(username='super_user', password='super_user')
+        record = RecordFactory(
+            auto_ptr=AutoPtrOptions.NEVER.id,
+            type='A',
+            name='blog.com',
+            content='192.168.1.0',
+            owner=self.regular_user1,
+        )
+        new_name = 'new-' + record.name
+        response = self.client.patch(
```
Comment (mkurek) [inline dnsaas/api/v2/tests.py:365]: could you make a test with patch without owner in data?

Comment (ghost) [inline dnsaas/api/v2/tests.py:365]: done

**Comment by mkurek** [comment]
:ok:

**Pull request merged**